lib/circular_percent_indicator.dart

```diff
@@ -253,7 +255,7 @@ class _CircularPercentIndicatorState extends State<CircularPercentIndicator>
                 backgroundColor: widget.backgroundColor,
                 startAngle: widget.startAngle,
                 circularStrokeCap: widget.circularStrokeCap,
-                radius: (widget.radius / 2) - widget.lineWidth / 2,
+                radius: widget.radius - widget.lineWidth / 2,
```

Comment:
avoid using widget.radius if we already have a `_diameter` variable

Reply:
@diegoveloper, 
I'll make necessary changes, but I don't understand motif as we already have radius then why do we again compute radius from (_diameter / 2)

Reply:
then we don't need a `_diameter` variable  after all, right?

Reply:
_diameter variable is created for Container ( as Container takes height & width ),
so given radius -> we compute diameter (as the only place where diameter is used is in Container size)
elsewhere -> radius itself is used

Reply:
_diameter can omitted if 
we replace it with `_radius * 2`

Reply:
you can use widget.radius*2 for the container

Reply:
but ok, I understand, that's fine then

Reply:
But I thought if its used at 2 places so if needed in Future case also so created new variable for ease !